src/VisualStudio/Core/Test.Next/Options/VisualStudioOptionStorageTests.cs

```diff
@@ -16,60 +16,158 @@ namespace Microsoft.CodeAnalysis.UnitTests;
 
 public class VisualStudioOptionStorageTests
 {
-    [Fact]
-    public void OptionValidation()
+    public static IEnumerable<object[]> ConfigNames
+    {
+        get
+        {
+            return OptionsTestInfo.CollectOptions(Path.GetDirectoryName(typeof(VisualStudioOptionStorage).Assembly.Location))
```

Comment:
>OptionsTestInfo.CollectOptions(Path.GetDirectoryName(typeof(VisualStudioOptionStorage).Assembly.Location)) [](http://example.com/codeflow?start=19&length=106)

Would it make sense to store this to static variable so that we don't need to do the reflection multiple times if all tests are running in the same run?

Reply:
I wasn't sure exactly how we would want to do that considering this is used both for test discovery and execution.

Reply:
Ah, I just need to set `DisableDiscoveryEnumeration`. Will update.